ghcide/src/Development/IDE/GHC/Compat/Outputable.hs

```diff
@@ -91,14 +96,9 @@ printSDocQualifiedUnsafe unqual doc =
     showSDocForUser unsafeGlobalDynFlags unqual doc
 #endif
 
-printSDocAllTheWay :: DynFlags -> SDoc -> String
+
 #if MIN_VERSION_ghc(9,2,0)
-printSDocAllTheWay dflags sdoc = renderWithContext ctxt sdoc
```

Comment:
Why this `#if` is still here? I mean, we can use the negated condition, cannot we?